pkg/plugins/policies/meshtls/api/v1alpha1/zz_generated.resource.go

```diff
@@ -145,7 +145,7 @@ var MeshTLSResourceTypeDescriptor = model.ResourceTypeDescriptor{
 	IsPolicy:                     true,
 	IsExperimental:               false,
 	SingularDisplayName:          "Mesh T L S",
-	PluralDisplayName:            "Mesh T L Ss",
+	PluralDisplayName:            "Mesh T L Ses",
```

Comment:
The way this is spaced out also seems like a bug to me no?

Reply:
It looks strange, I am not sure why.

Reply:
Ok seems related to pkg/core/resources/model/display_name.go
```
if unicode.IsUpper(c) && i != 0 {
   displayName += " "
}
```